react/features/base/participants/functions.js

```diff
@@ -65,9 +66,8 @@ export function getAvatarURL({ avatarID, avatarURL, email, id }: {
         if (urlPrefix) {
             urlSuffix = interfaceConfig.RANDOM_AVATAR_URL_SUFFIX;
         } else {
-            // Otherwise, use a default (meeples, of course).
-            urlPrefix = 'https://abotars.jitsi.net/meeple/';
-            urlSuffix = '';
+            urlPrefix = 'https://avatar-cdn.jitsi.net/';
+            urlSuffix = `/${_getInitials(name) || ' '}/200/avatar.png`;
```

Comment:
the service seems to be case sensitive: https://avatar-cdn.jitsi.net/lalala/Ss/200/avatar.png maybe we should capitalize the initials?

Reply:
That didn't even occur to me...I was indifferent to capitalization when testing. Sounds good to me. @yanas, can you confirm?

Reply:
Yep!

Reply:
I force pushed because I rebased onto latest master to get the pre-requisite commits. I also changed the string concatenation style a little bit for no defendable reason.